WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -1039,7 +1039,7 @@ class AbstractPostListViewController: UIViewController, WPContentSyncHelperDeleg
 
             if let error = error as NSError?, error.code == type(of: strongSelf).HTTPErrorCodeForbidden {
                 strongSelf.promptForPassword()
-            } else {
+            } else if let error = error {
```

Comment:
Are we sure we don't want to show an error if `error` is nil?

Reply:
You're right. It looks like I incorrectly analyzed `showXMLRPCErrorAlert`. It can accept a `nil` `error` since it will throw a generic alert message if it's `nil`. I'll change it back.